tools/wptserve/wptserve/request.py

```diff
@@ -348,12 +348,23 @@ def auth(self):
 class RequestHeaders(dict):
     """Dictionary-like API for accessing request headers."""
     def __init__(self, items):
-        for key, value in zip(items.keys(), items.values()):
-            key = key.lower()
-            if key in self:
-                self[key].append(value)
+        for header in items.keys():
+            key = header.lower()
+            # if there are other headers with this name, we need them
```

Comment:
can we get rid of the line by line comments?

maybe replace this one with `get all headers with the same name`